src/Api/TeamApi.php

```diff
@@ -16,4 +18,12 @@ public function getUser(): FetchAuthenticatedUserResponse
 
         return new FetchAuthenticatedUserResponse($data);
     }
+
+    public function getTeamDetails(string $team): ShowLoggedTeamAccountDetailsResponse
```

Comment:
As far as I know, the return type is not valid in PHP 7.3, which we still support, right?

Reply:
We do support PHP 7.3. However, AFAIK and according to a short research, this is supported...?